pkgs/development/tools/misc/ccache/default.nix

```diff
@@ -1,18 +1,23 @@
-{ stdenv, fetchurl, fetchpatch, runCommand, zlib, makeWrapper }:
+{ stdenv, fetchurl, fetchpatch, runCommand, perl, gcc, zlib, makeWrapper }:
```

Comment:
Is there a particular reason why gcc is a native build input? gcc should be part of stdenv.

Reply:
```perl``` too

Reply:
Without perl tests will fail with this error:
```
PASSED: 483 assertions, 96 tests, 10 suites
CC='gcc' /nix/store/zqh3l3lyw32q1ayb15bnvg9f24j5v2p0-bash-4.4-p12/bin/bash ./test/run
./test/run: line 51: perl: command not found
Compiler:         gcc ()
Compiler version: gcc (GCC) 7.3.0
./test/run: line 51: perl: command not found
./test/run: line 51: perl: command not found
CUDA compiler:    not available

Running test suite base../test/suites/base.bash: line 5: -c: command not found

FAILED

Test suite:     base
Test case:      Base case
Failure reason: Objects differ: reference_test1.o != test1.o

ccache -s:
cache directory                     /tmp/nix-build-ccache-3.4.1.drv-2/ccache-3.4.1/testdir.24043/.ccache
primary config                      /tmp/nix-build-ccache-3.4.1.drv-2/ccache-3.4.1/testdir.24043/ccache.conf
secondary config      (readonly)    
stats zero time                     Wed Feb 28 13:47:29 2018
cache hit (direct)                     0
cache hit (preprocessed)               0
cache miss                             1
cache hit rate                      0.00 %
cleanups performed                     0
files in cache                         1
cache size                           4.1 kB
max cache size                       5.0 GB

Test data and log file have been left in testdir.24043
make: *** [Makefile:106: test] Error 1
```

Reply:
`gcc` was removed from `nativeBuildInputs`.